project/Dependencies.scala

```diff
@@ -6,14 +6,16 @@ object Dependencies {
   val NettyIncubatorVersion         = "0.0.10.Final"
   val ScalaCompactCollectionVersion = "2.6.0"
   val ZioVersion                    = "1.0.12"
-
+  val sttpVersion                   = "3.3.18"
   val `scala-compact-collection` = "org.scala-lang.modules" %% "scala-collection-compat" % ScalaCompactCollectionVersion
   val netty                      = "io.netty"                % "netty-all"               % NettyVersion
   val `jwt-core`                 = "com.github.jwt-scala"   %% "jwt-core"                % JwtCoreVersion
-  val zio                        = "dev.zio"                %% "zio"                     % ZioVersion
-  val `zio-streams`              = "dev.zio"                %% "zio-streams"             % ZioVersion
-  val `zio-test`                 = "dev.zio"                %% "zio-test"                % ZioVersion % "test"
-  val `zio-test-sbt`             = "dev.zio"                %% "zio-test-sbt"            % ZioVersion % "test"
-  val `netty-incubator`          =
+  val sttp              = "com.softwaremill.sttp.client3" %% "core"                          % sttpVersion
+  val sttpzio           = "com.softwaremill.sttp.client3" %% "async-http-client-backend-zio" % sttpVersion
```

Comment:
`sttp` should only be a test dependency.

Reply:
Done